lib/index.ts

```diff
@@ -107,15 +98,15 @@ class PWMetrics {
     }
 
     if (resultHasExpectationErrors && this.flags.expectations) {
-      throw new Error(messages.getMessage('HAS_EXPECTATION_ERRORS'));
+      console.log(getMessage('HAS_EXPECTATION_ERRORS'));
```

Comment:
you want to let things continue in this case? im fine with that, just pointing out that the behavior is changing. :)
probably should console.error() if its staying non-throwing.